playwright/integrations/authenticated-chat-with-reconnect.spec.ts

```diff
@@ -152,4 +152,186 @@ test.describe('AuthenticatedChat @AuthenticatedChatWithChatReconnect', () => {
         expect(reconnectableChatsResponse.status()).toBe(200);
         expect(reconnectableChatsRequestHeaders['authenticatedusertoken']).toBe(authToken);
     });
+
+    test("ChatSDK.startChat() with a valid reconnect id should reconnect to previous chat", async ({ page }) => {
+        await page.goto(testPage);
+
+        const [_, chatTokenRequest, chatTokenResponse, sessionInitRequest, sessionInitResponse, runtimeContext] = await Promise.all([
+            await page.evaluate(async ({ omnichannelConfig, authUrl }) => {
+                const { OmnichannelChatSDK_1: OmnichannelChatSDK } = window;
+
+                const payload = {
+                    method: "POST"
+                };
+
+                const response = await fetch(authUrl, payload);
+                const authToken = await response.text();
+
+                const chatSDKConfig = {
+                    getAuthToken: () => authToken,
+                    chatReconnect: {
+                        disable: false,
+                    },
+                };
+
+                const chatSDK = new OmnichannelChatSDK.default(omnichannelConfig, chatSDKConfig);
+
+                await chatSDK.initialize();
+
+                await chatSDK.startChat();
+
+                location.reload();
```

Comment:
Is `location.reload()` really needed? We're already starting a new session via `page.evaluate()`

Reply:
No. Removed location.reload()